app/forms/reset_password_form.rb

```diff
@@ -47,6 +47,8 @@ def handle_valid_password
   end
 
   def update_user
+    @password_different = !user.valid_password?(password)
```

Comment:
Just a suggestion, I feel like we care about the true case more often? So I'd flip it, plus the name? but otherwise same same

```suggestion
    @password_matches_existing = user.valid_password?(password)
```

Reply:
Sounds good :+1: Updated in 7fe192bd23